OpenEdXMobile/res/values-es/strings.xml

```diff
@@ -208,6 +208,111 @@ Si no lo recibe en algunos minutos, asegúrese de revisar su carpeta de spam.</s
+  <!--Success Message after calendar for course added-->
+  <string name="message_after_course_calendar_added">Tu calendario de cursos ha sido añadido.</string>
+  <!--Title to remove calendar for course-->
+  <string name="title_remove_course_calendar">Remover el calendario \"{course_name}\"</string>
+  <!--Message to remove calendar for course-->
+  <string name="message_remove_course_calendar">¿Deseas eliminar el calendario de edX \"{course_name}\"?</string>
+  <!--Success Message after calendar for course removed-->
+  <string name="message_after_course_calendar_removed">Tu calendario de cursos ha sido eliminado.</string>
+  <!--Success Message after calendar for course update-->
+  <string name="message_after_course_calendar_updated">Tu calendario de cursos ha sido actualizado.</string>
+  <!--Title course calendar is out-of-date-->
+  <string name="title_calendar_out_of_date">Tu calendario de cursos está desactualizado</string>
+  <!--Message course calendar is out-of-date-->
+  <string name="message_calendar_out_of_date">Las fechas de tus cursos se han movido y tu calendario de cursos ya no está actualizado con tu nuevo horario.</string>
+  <!--Label My Courses Graded content Button-->
+  <string name="course_value_prop_button">Aprender más.</string>
+  <!--Label Course Modal heading-->
+  <string name="course_modal_heading">Cambia a la opción paga {course_name}</string>
+  <!--Label Course Modal features-->
+  <string name="course_modal_earn_certificate">Obtén un certificado de finalización verificado para mostrarlo en tu curriculum vitae.</string>
+  <string name="course_modal_unlock_access">Desbloquea el acceso a todas las actividades del curso, incluidas las tareas calificadas</string>
+  <string name="course_modal_full_access_course">Acceso total al contenido y al material del curso, incluso después de que finalice el curso</string>
+  <string name="course_modal_support_non_profit">Apoya nuestra misión sin fines de lucro en {platform_name}</string>
+  <!--Label Course Upgrade button-->
+  <!--Title Course Locked Modal-->
+  <string name="course_locked_modal_title">Obtén más información sobre funciones bloqueadas</string>
+  <!--Label Course Modal Unlock Graded Assignments-->
+  <string name="course_modal_unlock_graded_assignment"><u>Como desbloquear asignacianes calificadas</u></string>
+  <!--Label Course Modal Graded Assignments Locked-->
+  <string name="course_modal_graded_assignment_locked">Las tareas calificadas bloqueadas.</string>
+  <!--Label Course Modal Graded Assignments Locked description-->
+  <string name="course_modal_graded_assignment_locked_description">Cambia a la opción paga para obtener acceso a funciones bloqueadas como esta y aprovecha al máximo tu curso</string>
+  <!--Subtitle labels for video durations in hours-->
+  <!--Subtitle labels for video durations in minutes-->
+  <plurals name="video_duration_minutes">
+    <item quantity="one">%d</item>
+    <item quantity="other">%dminutos.</item>
```

Comment:
not sure, but IMO with one there would be a string like `%d min.` for one, and `%d minutos.` for other

Reply:
not sure either, I'll have to ask someone who speaks Spanish.

Reply:
I've updated the strings on Transifex after confirmation from a Spanish speaker.